src/test/java/io/kokuwa/maven/helm/UploadMojoTest.java

```diff
@@ -387,4 +440,59 @@ private void assertCatalog(UploadMojo mojo, Path archive, String uploadUrl) {
 		}
 
 	}
+
+	private void assertUploadVerifySuccess(UploadMojo mojo, RequestMethod method, String path) {
+		mockHelmShowChart();
+		assertDoesNotThrow(() -> mojo.execute(), "upload failed");
+		verifyRequests(mojo, method, path);
+	}
+
+	private void assertUploadVerifyFail(UploadMojo mojo, RequestMethod method, String path) {
+		mockHelmShowChart();
+		assertThrows(MojoExecutionException.class, mojo::execute, "could not verify");
+		verifyRequests(mojo, method, path);
+	}
+
+	private void mockHelmShowChart() {
+		mock.stubFor(WireMock.put(WireMock.urlMatching(".*"))
+				.willReturn(WireMock.ok()
+						.withStatus(200)));
+		mock.stubFor(WireMock.get(WireMock.urlMatching(".*/index\\.yaml$"))
+				.willReturn(WireMock.ok()
+						.withStatus(200)
+						.withHeader("Content-Type", "application/yaml")
+						.withBody(getIndexYamlBody())));
+		mock.stubFor(WireMock.get(WireMock.urlMatching(".*\\.tgz$"))
+				.willReturn(WireMock.ok()
+						.withStatus(200)
+						.withHeader("Content-Type", "application/gzip")
+						.withBodyFile("app-0.1.0.tgz")));
+	}
+
```

Comment:
this is the stubbing for what a "helm show chart app --version 0.1.0" call would expect to receive back from the remote